README.md

```diff
@@ -189,321 +173,17 @@ make install
 The setup is explained in
 [README\_docker.md](https://github.com/drand/drand/tree/master/docker/README_docker.md).
 
-### TLS setup: Nginx with Let's Encrypt
-Running drand behind a reverse proxy is the **default** method of deploying
-drand. Such a setup greatly simplify TLS management issues (renewal of
-certificates, etc). We provide here the minimum setup using
-[Nginx](https://www.nginx.com/) and
-[certbot](https://certbot.eff.org/instructions/) - make sure you have both
-binaries installed with the latest version; Nginx version must be at least >=
-1.13.10 for gRPC compatibility.
-
-+ First, add an entry in the Nginx configuration for drand:
-```bash
-# /etc/nginx/sites-available/default
-server {
-  server_name drand.nikkolasg.xyz;
-  listen 443 ssl http2;
- 
-  location / {
-    grpc_pass grpc://localhost:8080;
-  }
-  location /api/ {
-    proxy_pass http://localhost:8080;
-    proxy_set_header Host $host;
-  }
-}
-```
-**Note**: you can change 
-1. the port on which you want drand to be accessible by changing the line
-   `listen 443 ssl http2` to use any port.
-2. the port on which the drand binary will listen locally by changing the line
-   `proxy_pass http://localhost:8080; ` and ` grpc_pass grpc://localhost:8080;`
-   to use any local port.
-
-+ Run certbot to get a TLS certificate:
-```bash
-sudo certbot --nginx
-```
-
-+ **Running** drand now requires to add the following options:
-```bash
-drand start --tls-disable --listen 127.0.0.1:8080
-```
-
-The `--listen` flag tells drand to listen on the given address instead of the
-public address generated during the setup phase (see below).
-
-## Usage
 
-This section explains in details the workflow to have a working group of drand
-nodes generate randomness. On a high-level, the workflow looks like this:
-+ **Setup**: generation of individual long-term key pair and the group file and
-  starting the drand daemon.
-+ **Distributed Key Generation**: each drand node collectively participates in
-  the DKG.
-+ **Randomness Generation**: the randomness beacon automatically starts as soon
-  as the DKG protocol is finished.
+### Usage
```

Comment:
I think it's correct to highlight the usage _after_ setup, but we do need a link here to the document with setup instructions in it for people/devs trying to run their own nodes.